crates/codegen/schema/src/types/scanner.rs

```diff
@@ -16,13 +16,6 @@ pub enum ScannerDefinition {
     #[schemars(title = "Choice Expression")]
     Choice(Vec<ScannerRef>),
 
```

Comment:
Not sure I understand why `DelimitedBy` and `SeparatedBy` were deleted. My understanding is that they are for future error recovery, same as the parser ones, which this PR leaves intact.

Reply:
f2f: to simplify the code, because we don't/won't be error correcting within tokens.